exercises/perfect-numbers/divisors.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var Divisors = function() {
+
+}
```

Comment:
need a semicolon here, and a few other places as well.  Let me know if you need me to call them out.